.github/dependabot.yml

```diff
@@ -40,6 +40,7 @@ updates:
       - dependency-name: io.quarkus.*:*
       - dependency-name: io.quarkus:*
       # Elytron
+      - dependency-name: org.wildfly.common:*
```

Comment:
This should be before the `# Elytron` comment, which is referring to `org.wildfly.security`.

Reply:
Yeah, I thought about changing it to WildFly and then forgot. I moved it in the misc section at the top.